streaming/src/test/scala/org/apache/spark/streaming/ReceiverSuite.scala

```diff
@@ -295,6 +334,27 @@ class ReceiverSuite extends TestSuiteBase with Timeouts with Serializable {
   }
 
   /**
+   * An implementation of Task running a NetworkReceiver used for testing whether
+   * we can start/stop receiver properly.
+   */
+  class FakeReceiverTask(
+    stageId: Int,
+    executor: FakeReceiverSupervisor,
+    prefLocs: Seq[TaskLocation] = Nil) extends Task[Unit](stageId, 0) {
```

Comment:
It looks like we never call this with a non-Nil value of `prefLocs`, so we can probably drop the constructor argument and just hardcode `Nil` into the `preferredLocations` method.